### PR TITLE
Add WiFi section to the OTA tutorial.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: docs/tutorials/misc/ota.mdx [hunk 1]
```diff
@@ -257,6 +257,22 @@ the device through Jaguar to get the Jaguar service back.
...
 
+## WiFi
+Depending on how you have configured the network on your device, you might
+need to bake the WiFi credentials into the OTA image. This can be done
+by adding a config file with the credentials to the extraction step.
+
+``` makefile
+ota.bin: validate.snapshot firmware.envelope
+	$(TOIT_FIRMWARE) -e firmware.envelope container install validate validate.snapshot
+	echo '{ "wifi": { "wifi.ssid": "myssid", "wifi.password": "mypassword" } }' > wifi.json
```
Comment (kasperl) [inline docs/tutorials/misc/ota.mdx:268]: I think you can also do this through:

```
$(TOIT_FIRMWARE) -e firmware.envelope property set wifi '{"wifi.ssid": "myssid", "wifi.password": "mypassword"}'
```

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: docs/tutorials/misc/ota.mdx (hunk 1 above)
Comment (floitsch) [reply docs/tutorials/misc/ota.mdx]: done.

File: docs/tutorials/misc/ota.mdx [hunk 2]
```diff
@@ -257,6 +257,24 @@ the device through Jaguar to get the Jaguar service back.
...
+need to bake the WiFi credentials into the OTA image. This can be done
+by adding a config file with the credentials to the extraction step.
+
+``` makefile
+WIFI_CREDENTIALS := {"wifi.ssid":"<my-ssid>","wifi.password":"<my-password>"}
+
+ota.bin: validate.snapshot firmware.envelope
+	$(TOIT_FIRMWARE) -e firmware.envelope container install validate validate.snapshot
+	$(TOIT_FIRMWARE) -e firmware.envelope property set wifi '$(WIFI_CREDENTIALS)'
+	$(TOIT_FIRMWARE) -e firmware.envelope extract --format=binary -o ota.bin --config wifi.json
```
Comment (kasperl) [inline docs/tutorials/misc/ota.mdx:271]: You can drop the config here now.

Comment (floitsch) [reply docs/tutorials/misc/ota.mdx:271]: thanks.
done.

**Pull request merged**